.github/support.yml

```
@@ -6,7 +6,7 @@ supportLabel: support
 # Comment to post on issues marked as support requests. Add a link
 # to a support page, or set to `false` to disable
 supportComment: |
-  👋 Thanks for using Material-UI!
+  👋 Thanks for using MUI!
```

Should we differentiate the products? 
```suggestion
  👋 Thanks for using MUI Core!
```